app/main.py

```diff
@@ -1,3 +1,59 @@
+from typing import Union, Type
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: (int, float)) -> None:
+        self.km = km
+
+    @staticmethod
+    def validator(
+            other: Union["Distance", int, float],
```

Comment:
```suggestion
            other: Union[Distance, int, float],
```
fix everywhere

Reply:
When I do so I have an error:
<img width="766" alt="image" src="https://github.com/mate-academy/py-distance-class/assets/10523657/1bba72d8-b223-4611-8ed4-870898bd7e56">

Reply:
Added `from __future__ import annotations` and now it's ok.